<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditComputer</class>
 <widget class="QDialog" name="EditComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="error_name">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>180</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Built</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_wasbuilt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="error_year">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_built">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_year">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="error_type">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_type">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="table_computer">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Built</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
